<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1400</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="add_textbook_background_frame">
     <property name="minimumSize">
      <size>
       <width>1400</width>
       <height>900</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1400</width>
       <height>900</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_20">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_21">
        <property name="minimumSize">
         <size>
          <width>1000</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLineEdit" name="author_line_edit">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>80</y>
           <width>311</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>40</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Category</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>80</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Author</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>110</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Title</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>150</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Edition</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>180</y>
           <width>161</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Textbook file source path</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>210</y>
           <width>161</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Solutions manual file source path</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="title_line_edit">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>110</y>
           <width>311</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="category_combo_box">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>40</y>
           <width>311</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="num_chapters_spin_box">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>270</y>
           <width>42</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_17">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>270</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Number of Chapters</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="sections_check_box">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>300</y>
           <width>251</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Are the chapter exercises grouped by sections?</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="eoc_check_box">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>320</y>
           <width>251</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Are there an end of chapter/review section?</string>
         </property>
        </widget>
        <widget class="QTableWidget" name="num_sections_table">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>350</y>
           <width>411</width>
           <height>291</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="tb_file_path_line_edit">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>180</y>
           <width>381</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="sm_file_path_line_edit">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>210</y>
           <width>381</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="tb_file_path_browse_button">
         <property name="geometry">
          <rect>
           <x>760</x>
           <y>180</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
        <widget class="QPushButton" name="sm_file_path_browse_button">
         <property name="geometry">
          <rect>
           <x>760</x>
           <y>210</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="edition_spin_box">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>150</y>
           <width>42</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="exit_add_textbook_page_button">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;-</string>
         </property>
        </widget>
        <widget class="QPushButton" name="submit_button">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>660</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="console_text_edit">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>720</y>
           <width>771</width>
           <height>151</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="clear_button">
         <property name="geometry">
          <rect>
           <x>860</x>
           <y>670</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_22">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
